src/path/mod.rs

```diff
@@ -311,7 +311,7 @@ fn get_target_prefix(path: &str) -> (PathPrefix, &str) {
 }
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
-#[cfg_attr(feature = "proptest", derive(proptest_derive::Arbitrary))]
+#[cfg_attr(any(test, feature = "proptest"), derive(proptest_derive::Arbitrary))]
```

Comment:
Do we want `all` vs `any`?

Reply:
As above, I enabled proptests for testing, so these features are enabled for both `cargo test` and when the `proptest` feature is enabled by another crate (i.e. Vector).